docs/developers_guide/quick_start.rst

```diff
@@ -94,14 +94,19 @@ See the machine under :ref:`dev_supported_machines` for a list of available
 compilers to pass to ``-c``.  If you don't supply a compiler, you will get
 the default one for that machine (usually Intel). Typically, you will want the
 default MPI flavor that compass has defined for each compiler, so you should
-not need to specify which MPI version to use but you may do so with ``-i`` if
-you need to.
+not need to specify which MPI version to use but you may do so with ``--mpi``
+if you need to.
 
 If you are on a login node, the script should automatically recognize what
 machine you are on.  You can supply the machine name with ``-m <machine>`` if
 you run into trouble with the automatic recognition (e.g. if you're setting
 up the environment on a compute node, which is not recommended).
 
+If you are working with MALI, you should specify ``--with_albany``.  This will
```

Comment:
Do you want to add that it's safe to add this option for MPAS-Ocean but it's recommended not to do so unless a user wants to be able to run both models with the same COMPASS env?

Reply:
Yes, perfect! I'll add that.